doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +169,121 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
+     any other desired AWS location.
+   * ``Admin Ingress Location`` defines the locations from which you
+     can access this cluster as an administrator. Enter ``0.0.0.0/0``
+     for the most permissive approach.
+   * ``SSH Key`` is a dropdown list of keys attached to your account.
+     The one you created in Step 1 should be listed here. This will allow
+     you to SSH into the machines if you desire.
+   * ``Node Capacity`` defines the number of machines you've got available.
+     This will depend on the ``Instance Type`` that you choose. E.g., if you
+     want each user to have 2GB and you expect 10 users, choose a combination
+     of ``Instance Type`` and ``Node Capacity`` that meets this requirement.
+   * ``Instance Type`` defines what kind of machine you're requesting. See
+     this `list of instance types with Amazon <https://aws.amazon.com/ec2/instance-types/>`_
+     as well as this list of `pricing for each instance type <https://aws.amazon.com/ec2/pricing/on-demand/>`_.
+   * ``Disk Size`` corresponds to the hard disk for each node. Note that this is
+     different from the disks that users will use for their own notebooks/data.
+     This disk should be large enough to contain the size of any Docker
+     images you're serving with the JupyterHub.
+   * ``Instance Type (Bastion Host)`` corresponds to a computer that allows
+     for easy SSH access to your Kubernetes cluster. This does not need to
+     be a fancy computer. You may leave these as defaults. For more information
+     on the Bastion Host, `see here <http://docs.aws.amazon.com/quickstart/latest/linux-bastion/architecture.html>`_.
+
+   **Enter AWS instance information (page 2)**: On the second page you may leave
+   all of these fields as is or customize as you wish. When done, hit ``Next``. Then
+   confirm and hit ``Next`` once more.
+
+   AWS will now create the computational resources defined in the Heptio
+   template (and according to the options that you chose). This will
+   take a few minutes.
+
+   To see the status of the resources you've requested,
+   see the ``CloudFormation`` page. You should see two stacks being created,
+   each will have the name you've requested. When they're done creating,
+   continue with the guide.
+
+3. Follow Step 3 of the `Heptio guide`_. Install ``kubectl``,
+   and test that your new Kubernetes cluster works properly. In this step
+   you'll need the SSH key file that was generated in Step 1.
+
+4. Create a file, ``storageclass.yml`` on your local computer, and enter
+   this text::
+
+       kind: StorageClass
+       apiVersion: storage.k8s.io/v1
+       metadata:
+         annotations:
+            storageclass.beta.kubernetes.io/is-default-class: "true"
+         name: gp2
+       provisioner: kubernetes.io/aws-ebs
+       parameters:
+         type: gp2
+
+   Next, run this command:
+
+       .. code-block:: bash
+
+          kubectl apply -f storageclass.yml
```

Comment:
I reinstalled kubectl and still seeing the same message. Here is what I get for `kubectl version`

```
Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.3", GitCommit:"2c2fe6e8278a5db2d15a013987b53968c743f2a1", GitTreeState:"clean", BuildDate:"2017-08-03T07:00:21Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.2", GitCommit:"922a86cfcd65915a9b2f69f3f193b8907d741d9c", GitTreeState:"clean", BuildDate:"2017-07-21T08:08:00Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}

```